nvm.sh

```diff
@@ -1935,6 +1935,12 @@ nvm_install_binary() {
     return 0
   fi
 
+  local nosource
+  # Read nosource from arguments
+  if [ $nosource -eq 1]; then
```

Comment:
@ljharb What would be the best way to read the `-b` flag here. I could not see the `-s` flag being read anywhere. I just put a boolean here and thought that we could read the flag and set it accordingly

Reply:
the `-s` flag is read here: https://github.com/nvm-sh/nvm/blob/master/nvm.sh#L2901-L2903

and you'd need to pass it into `nvm_install_binary` here: https://github.com/nvm-sh/nvm/blob/master/nvm.sh#L3168 as either an arg, or an env var.